src/batch-client/index.ts

```diff
@@ -301,6 +301,15 @@ export class ZimbraBatchClient {
 			body: options
 		});
 
+	public dismissCalendarItem = (appointment: any, task: any) =>
+		this.jsonRequest({
+			name: 'DismissCalendarItemAlarm',
+			body: {
+				appt: appointment,
+				task
+			}
+		}).then(res => !!res);
```

Comment:
To cast to a boolean without the extra function declaration you can use the `Boolean` constructor itself: `.then(Boolean)`.